README.md

```diff
@@ -1,12 +1,16 @@
 <img src="https://travis-ci.org/innolitics/hdat.svg?branch=master">
 
-# High Dimensional Algorithm Testing Tool (HDATT)
+# High Dimensional Algorithm Testing tool (HDAT)
```

Comment:
What do you think about "High Dimensional Algorithm Tester"?

Reply:
That's perfect!